index.html

```diff
@@ -24,9 +24,9 @@ <h1 class="neon-text">G I M E L</h1>
   </div>
     <div class="space2"></div>
     <div class="space1">
-        <p class="neon-paragraph wide-element">The Polish CTF team.</p>
+        <p class="neon-paragraph wide-element">A Polish CTF team.</p>
```

Comment:
this is controversial - it was supposed to imply that we are the only real polish ctf team (i.e. just ctf doesn't exist)

Reply:
Fair enough. Let's become "The" Polish CTF team.